Ines/indx.js

```diff
@@ -0,0 +1,20 @@
+//1
+let determiner = 2;
+console.log("1==>",determiner<0?"Less than 0":"Greater or equal to 0")
```

Comment:
In line 3 you also have to make a new variable called x ans assign a value to x depending on the value of determined.